clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -59,6 +59,7 @@ export const ConsentButtons = ({
                 data-testid={`fides-i18n-option-${lang.locale}`}
                 onClick={() => handleLocaleSelect(lang.locale)}
                 isActive={currentLocale === lang.locale}
+                title={lang.label_en}
```

Comment:
why is this line added?

Reply:
I was thinking it might be nice to have the english version of the language translations as a tooltip when you hover over each language but forgot to add that before. Overkill?

Reply:
ohhhh I see haha. We'll just want to confirm with @mfbrown

Reply:
I'll pull it out of this PR and add it to the new story he just created for me that's more related. I'll ask him about it there :) https://ethyca.atlassian.net/browse/PROD-1908